spec/pom.xml

```diff
@@ -16,65 +16,67 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.6</version>
-        <relativePath/>
+        <relativePath />
     </parent>
-    <modelVersion>4.0.0</modelVersion>
+
     <groupId>jakarta.stl</groupId>
     <artifactId>jakarta-stl-spec</artifactId>
-    <packaging>pom</packaging>
     <version>2.0-SNAPSHOT</version>
-    <name>Jakarta Standard Tag Library Specification</name>
+    <packaging>pom</packaging>
 
-    <properties>
-        <site.output.dir>${project.build.directory}/staging</site.output.dir>
-        <maven.site.skip>true</maven.site.skip>
-        <asciidoctor.maven.plugin.version>1.5.7.1</asciidoctor.maven.plugin.version>
-        <asciidoctorj.version>1.6.2</asciidoctorj.version>
-        <asciidoctorj.pdf.version>1.5.0-alpha.16</asciidoctorj.pdf.version>
-        <jruby.version>9.2.6.0</jruby.version>
-        <!-- status: DRAFT, BETA, etc., or blank for final -->
-        <status>Final Release</status>
-        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
-        <revisiondate>${maven.build.timestamp}</revisiondate>
-    </properties>
+    <name>Jakarta Standard Tag Library Specification</name>
 
     <scm>
         <connection>scm:git:git@github.com:eclipse-ee4j/jakarta-stl-spec.git</connection>
         <developerConnection>scm:git:git@github.com:eclipse-ee4j/jakarta-stl-spec.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jakarta-stl-spec</url>
         <tag>HEAD</tag>
     </scm>
-
     <distributionManagement>
         <site>
             <url>scm:git:git@github.com:eclipse-ee4j/jakarta-stl-spec.git</url>
         </site>
     </distributionManagement>
 
+    <properties>
+        <site.output.dir>${project.build.directory}/staging</site.output.dir>
+        <maven.site.skip>true</maven.site.skip>
+        <asciidoctor.maven.plugin.version>1.5.7.1</asciidoctor.maven.plugin.version>
+        <asciidoctorj.version>2.3.0</asciidoctorj.version>
+        <asciidoctorj.pdf.version>1.5.3</asciidoctorj.pdf.version>
+        <jruby.version>9.2.11.1</jruby.version>
+        <!-- status: DRAFT, BETA, etc., or blank for final -->
+        <status>Final Release</status>
```

Comment:
I have this as DRAFT in my PR so we might get a merge conflict here that we'll need to resolve in on of these PRs.

Reply:
Yes, I'll resolve the merge conflict. Should be easy.